engine-tests/src/test/java/org/terasology/math/BorderTest.java

```diff
@@ -33,46 +21,46 @@ public void initBorder() {
 
     @Test
     public void shrinkSameBorderAndRegionSize() {
-        assertEquals(border.shrink(Rect2i.createFromMinAndSize(10, 10, 10, 10)), Rect2i.EMPTY);
+        assertEquals(border.shrink(JomlUtil.from(Rect2i.createFromMinAndSize(10, 10, 10, 10))), JomlUtil.from(Rect2i.EMPTY));
```

Comment:
`RectUtility::createFromMinAndSize` and in turn `Border#shrink` can produce invalid `Rectangle2i` (e.g, by `RectUtility.createFromMinAndSize(20, 20, -10, -10)`). I'm not sure where/how we want to fix this... @pollend ?

Reply:
A potential fix for `Border#shrink` might be:

```java
    public Rectanglei shrink(Rectanglei region) {
        final Rectanglei shrinked = RectUtility.createFromMinAndSize(region.minX + getLeft(), region.minY + getTop(),
                region.lengthX() - getTotalWidth(), region.lengthY() - getTotalHeight());
        if (shrinked.isValid()) {
            return shrinked;
        } else {
            return new Rectanglei();
        }
    }
```